### PR TITLE
Create readme for the 3.0 version of the OAS

### PR DESCRIPTION
**Pull request by usarid**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,62 @@
...
+## Tools and Libraries
+
+Looking to see how you can create your own OpenAPI definition, present it or otherwise use it? Check out the growing
+[list of 3.0 Implementations](IMPLEMENTATIONS.md).
 
 ## Participation
 
-The OpenAPI Specification is a community driven, open project hosted by the Linux Foundation. The OAI encourages participation from individuals and companies alike. If you want to participate in the evolution of the OpenAPI Specification, consider taking the following actions:
+The current process for development of the OpenAPI Specification is described in 
+[Development Guidelines](DEVELOPMENT.md).
```
Comment (fehguy) [inline README.md]: It's good to leave this, but we will need to update this doc

File: README.md [hunk 2]
```diff
@@ -1,31 +1,62 @@
...
+The OpenAPI Specification is a community driven, open specification within the [Open API Initiative](https://www.openapis.org/), a Linux Foundation Collaborative Project.
 
-The current, released version of the OpenAPI Specification is 2.0, through donation of the Swagger Specification to the OAI by SmartBear Software.  If you are interested in the release specification, see the [master branch](https://github.com/OAI/OpenAPI-Specification/blob/master/README.md) of this project.
+The goal of the OpenAPI Specification is to define a standard, language-agnostic interface description for REST APIs which allows both humans and computers to discover and understand the capabilities of the service without access to source code, documentation, or through network traffic inspection.  When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic.  Similar to what interface descriptions have done for lower-level programming, the OpenAPI Specification removes the guesswork in calling the service.
```
Comment (DavidBiesack) [inline README.md]: Suggest: "The OpenAPI Specification defines ..."

Comment (DavidBiesack) [inline README.md]: Suggest: "without access to source code, additional documentation," (the OpenAPI definition *is* documentation)

File: README.md [hunk 3]
```diff
@@ -1,31 +1,62 @@
...
 
-Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the OpenAPI Specification, this branch will be merged to master.
+Use cases for machine-readable API interfaces include interactive documentation, code generation for documentation, client, and server, as well as automated test cases. OpenAPI descriptions describe APIs via YAML or JSON documents that adhere to the OpenAPI Specification documented in this repository.  These documents can either be produced and served statically, or be generated dynamically from your application.
```
Comment (DavidBiesack) [inline README.md]: Suggest: "Use cases for machine-readable API interfaces include interactive documentation; code generation for documentation, client, and server; automated test cases."

Comment (DavidBiesack) [inline README.md]: Remove "in this repository" - that is, the document is the spec (the fact that it is in a GigHub repository is unimportant)

File: README.md [hunk 4]
```diff
@@ -1,31 +1,62 @@
...
 
-The current process for development of the OpenAPI Specification is described in [Development Guidelines](https://github.com/OAI/OpenAPI-Specification/blob/OpenAPI.next/DEVELOPMENT.md).
+Without going into a long history of interfaces to Web Services, this is not the first attempt to do so.  We can learn from CORBA, WSDL and WADL.  These specifications had good intentions but were limited by proprietary vendor-specific implementations, being bound to a specific programming language, and goals which were too open-ended.  In the end, they failed to gain traction.
```
Comment (DavidBiesack) [inline README.md]: Suggestion: replace the first sentence with "The OpenAPI Specification design was informed by lessons learned from CORBA, WSDL, WADL, and other historical efforts in defining web services. These other specifications ..."

(I'm not sure one can claim CORBA or WSDL did not gain traction - they did, but were not easy to use. Also, this para seems more related to REST vs other web service architecture, so feels misplaced.)

File: README.md [hunk 5]
```diff
@@ -1,31 +1,62 @@
...
+
+The OpenAPI Specification does not require you to rewrite your existing API.  It does not require binding any software to a service--the service being described may not even be yours.  It does, however, require the capabilities of the service be described in the structure of the OpenAPI Specification.  Not all services can be described by OpenAPI--this specification is not intended to cover every possible use-case of a REST API. The OpenAPI Specification does not define a specific development process such as design-first or code-first.  It does facilitate either technique by establishing clear interactions with a REST API.
+
+This GitHub project is the starting point for OpenAPI.
+Here you will find the information you need about the OpenAPI Specification, simple examples of what it looks like,
+and some general information regarding the project.
+
+## Current Version - 3.0
+
+The current version of the OpenAPI specification is 3.0 - and you can find it [here](versions/3.0.md).
```
Comment (DavidBiesack) [inline README.md]: Suggest: `The current version of the OpenAPI specification is [OpenAPI Specification 3.0](versions/3.0.md).`

File: README.md [hunk 6]
```diff
@@ -1,31 +1,62 @@
...
+
+### [Previous Versions](versions/2.0.md)
```
Comment (DavidBiesack) [inline README.md]: defer to the link to 2.0 to the paragraph, not the header.

File: README.md [hunk 7]
```diff
@@ -1,31 +1,62 @@
...
+
+This repository also contains the OpenAPI Specification 2.0, which is identical to the Swagger 2.0 specification,
```
Comment (DavidBiesack) [inline README.md]: `[OpenAPI Specification 2.0](versions/2.0)`

File: README.md [hunk 8]
```diff
@@ -1,31 +1,62 @@
...
+
+Looking to see how you can create your own OpenAPI definition, present it or otherwise use it? Check out the growing
+[list of 3.0 Implementations](IMPLEMENTATIONS.md).
 
 ## Participation
 
-The OpenAPI Specification is a community driven, open project hosted by the Linux Foundation. The OAI encourages participation from individuals and companies alike. If you want to participate in the evolution of the OpenAPI Specification, consider taking the following actions:
+The current process for development of the OpenAPI Specification is described in 
+[Development Guidelines](DEVELOPMENT.md).
+Development of the next version of the OpenAPI Specification is guided by the [OAI Technical Contributors Board](CONTRIBUTORS.md).  This group of committers bring their API expertise, incorporate feedback from the community, and expand the group of committers as appropriate.  All development activity on the future specification will be performed as features and merged into this branch.  Upon release of the future specification, this branch will be merged to master.
```
Comment (usarid) [inline README.md]: Needs to get updated with TDC language

Comment (DavidBiesack) [inline README.md]: `[Technical Developer Community](https://www.openapis.org/participate/how-to-contribute/governance#TDC)`, not OAI Technical Contributors Board

Comment (olensmar) [inline README.md]: I suggest replacing "OAI Technical Contributors Board" with simply "TDC Contributors" - it's important to note that the OAI or its official members have no direct influence or mandate on the work/direction of the TDC  - the suggested wording could be interpreted differently.

**Comment by olensmar** [comment]
@usarid i'd be happy to help make the suggested changes - but your branch doesn't seem to be available...

**Comment by usarid** [comment]
Thanks @olensmar I've been too slammed to look into what happened here, but looking at it now.

File: README.md (hunk 8 above)
Comment (usarid) [inline README.md]: Updated in the gdoc

File: README.md (hunk 1 above)
Comment (usarid) [inline README.md]: That should be a different PR - who can take that?

File: README.md (hunk 2 above)
Comment (usarid) [inline README.md]: Thanks, incorporated into the gdoc

File: README.md (hunk 3 above)
Comment (usarid) [inline README.md]: Thanks, incorporated into the gdoc

File: README.md (hunk 4 above)
Comment (usarid) [inline README.md]: I'm going to omit the paragraph entirely. I don't believe it belongs in the readme of the spec, maybe more in commentary somewhere else.

File: README.md (hunk 5 above)
Comment (usarid) [inline README.md]: Thanks, incorporated into the gdoc.

File: README.md (hunk 6 above)
Comment (usarid) [inline README.md]: Thanks, incorporated into the gdoc.

File: README.md (hunk 7 above)
Comment (usarid) [inline README.md]: Thanks, incorporated into the gdoc.

File: README.md (hunk 8 above)
Comment (usarid) [inline README.md]: Thanks, incorporated a slight modification of this in the gdoc.

File: README.md (hunk 7 above)
Comment (DavidBiesack) [inline README.md]: Oops, sorry.... that should be `[OpenAPI Specification 2.0](versions/2.0.md)`  (i.e. the `.md` extension is required)

**Comment by usarid** [comment]
This was eventually implemented in https://github.com/OAI/OpenAPI-Specification/pull/1281 so let's close this one.
